packages/shared/src/components/filters/TagItemList.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { ReactElement } from 'react';
 import { Tag } from '../../graphql/feedSettings';
 import { TagActionArguments } from '../../hooks/useTagAndSource';
-import { FiltersList } from './common';
+import { FiltersList, FiltersGrid } from './common';
 import TagItemRow from './TagItemRow';
 
 type TagItemListProps = {
+  version?: string;
```

Comment:
Good point! Let's see, we were thinking of replacing this with a boolean value as the `version` seems to be a bit unfit, but if in case, I agree, we should refactor to using enum.